gulpfile.js

```diff
@@ -112,10 +112,10 @@ gulp.task('default', debugBuild);
 
 // Get platform from commandline args
 // #
-// # gulp <task> [<platform>]+        Run only for platform(s) (with <platform> one of --linux64, --linux32, --armv7, --osx64, --win32, --win64, --chromeos or --android)
+// # gulp <task> [<platform>]+        Run only for platform(s) (with <platform> one of --linux64, --linux32, --armv7, --osx64, --win32, --win64, or --android)
 // #
 function getInputPlatforms() {
-    const supportedPlatforms = ['linux64', 'linux32', 'armv7', 'osx64', 'win32','win64', 'chromeos', 'android'];
+    const supportedPlatforms = ['linux64', 'linux32', 'armv7', 'osx64', 'win32','win64', 'android'];
```

Comment:
```suggestion
    const supportedPlatforms = ['linux64', 'linux32', 'armv7', 'osx64', 'win32', 'win64', 'android'];
```

Reply:
Hopefully implementing linter will suppress my pointless reviews complaining about styling issues :trollface:

Reply:
I don't think they are pointless, I like to have the code right. I will change it tomorrow. Thanks for the review.

Reply:
Sorry @haslinghuis, I should have expressed my thoughts in better way 😉

Reply:
no worries 😃